D.py

```diff
@@ -0,0 +1,2 @@
+def goodmornig():
```

Comment:
```suggestion
def goodmorning():
```
goodmornig →goodmorning

Reply:
了解しました。